### ISSUE TITLE
Suggested improvements for the "/namespace" endpoint

### ISSUE THREAD

**Issue by shea256** [blockstack/blockstack-resolver#96]
I noticed you added ".id" at the end of all the names in the "/namespace" endpoint. Nice ;) And the stats metric is awesome. Excited to be able to use that across sites.

As far as improvements to the endpoint, I'd like to suggest we update the URL to the following:

```
/v2/namespaces/id
```

I'd also update the field in the response from "usernames" to "names" (to be more generic and inclusive across all namespaces).

Next... if we ever have an endpoint that lists all names across all namespaces, I would have that URL look like this:

```
/v2/namespaces
```

...and then have that return the following:

```json
{
  "namespaces": [
    { "tld": "id", "stats": { "registrations": 46180 }, "names": [] }
  ]
}
```

Not sure if "tld" is the right field name but I couldn't think of anything else. Any suggestions.

If we don't plan on adding this unified endpoint, then that's fine. The only thing we'll probably need is a counter that adds up all names across namespaces.

**Comment by muneeb-ali** [comment]
Good suggestions. Can't think of anything better than tld. 'namespace' is one option.

I like having the united endpoint and that's the one we should probably display to people, so it's clear that there are multiple namespaces and we can count across all of them.

**Comment by shea256** [comment]
Yeah, you're right. Good call.

**Comment by muneeb-ali** [comment]
So this format display 'names' before other fields and you can't really see the other fields.

Check out http://resolver.onename.com/v2/namespaces

Trying to use 'info' so that the information is displayed above the list of names.

**Comment by shea256** [comment]
Oh snap, I just discovered something:

```python
>>> import json
>>> import collections
>>> json.encoder.c_make_encoder = None
>>> d = collections.OrderedDict([("namespace", "id"), ("registrations", 46180), ("names", [])])
>>> print json.dumps(d, indent=2)
{
  "namespace": "id", 
  "registrations": 46180, 
  "names": []
}
```

I got this idea from here: http://stackoverflow.com/a/4402799/1530754

**Comment by shea256** [comment]
No longer will we be at the whims of or subjected to the tyranny of the default JSON sorting!

**Comment by muneeb-ali** [comment]
That worked. However, Flask's jsonify function sorts by default and had to configure Flask to ignore this. Not very well documented and hidden in an issue: https://github.com/mitsuhiko/flask/issues/974

**Comment by shea256** [comment]
Nice! And hm, yeah that's strange they do that.

**Comment by larrysalibra** [comment]
> As far as improvements to the endpoint, I'd like to suggest we update the URL to the following:

> /v2/namespaces/id

I was trying to use `/v2/namespaces/id` too before I looked at the code and noticed it doesn't exist. 
+1 for this

**Comment by larrysalibra** [comment]
`/v2/namespaces` only seems to only work on https://resolver.onename.com. I'm running the the latest commit (ce062a85a65a2fbc2cae0e08c9505bb5a4031074) on my resolver.

I only see a route for `/v2/namespace` https://github.com/blockstack/blockstack-resolver/blob/ce062a85a65a2fbc2cae0e08c9505bb5a4031074/resolver/server.py#L283

The `/v2/namespace` also behaves differently on https://resolver.onename.com compared to the code in this repo as well. Did the code not get checked in to the public repo? 

<img width="1038" alt="screen shot 2016-03-22 at 6 27 57 pm" src="https://cloud.githubusercontent.com/assets/597182/13948683/5a22483e-f05c-11e5-843a-fc6e949da5ce.png">
<img width="1043" alt="screen shot 2016-03-22 at 6 27 39 pm" src="https://cloud.githubusercontent.com/assets/597182/13948688/5e14b2ce-f05c-11e5-9b68-83adbbecb893.png">

**Comment by muneeb-ali** [comment]
Fixed in fe0add7586dabc61fe9bce6a80ac90516fac2a10
